src/ansys/meshing/prime/examples/examples.py

```diff
@@ -901,13 +907,14 @@ def download_pcb_scdoc(
 
     Parameters
     ----------
-    destination: Optional[str]
-        Destination for the file to be downloaded.
-        If nothing is provided, the default path in app data is used.
-    force: bool
-        Option to download the file.
-        If true, the file is always downloaded.
-        If false, an existing file in the cache may be reused.
+    destination : str, optional
+        Path to which you download the example file. The default
```

Comment:
```suggestion
        Path to download the example file to. The default
```
It's OK to end sentence with a preposition, especially to improve readability.